simpleansi.go

```diff
@@ -16,7 +16,29 @@ func MoveCursor(row, col int) {
 	fmt.Printf("\x1b[%d;%df", row+1, col+1)
 }
 
+const RESET = "\x1b[0m"
+
+var COLOURS = map[string]string{
+	"BLACK":   "\x1b[1;30;40m",
+	"RED":     "\x1b[1;31;41m",
+	"GREEN":   "\x1b[1;32;42m",
+	"BROWN":   "\x1b[1;33;43m",
+	"BLUE":    "\x1b[1;34;44m",
+	"MAGENTA": "\x1b[1;35;45m",
+	"CYAN":    "\x1b[1;36;46m",
+	"GREY":    "\x1b[1;37;47m",
+}
+
 // WithBlueBackground wraps the given text with blue background and reset escape sequences.
 func WithBlueBackground(text string) string {
-	return "\x1b[44m" + text + "\x1b[0m"
+	return "\x1b[44m" + text + RESET
+}
+
+// WithBackground wraps the given 'text' with 'colour' background and reset escape sequences.
+func WithBackground(text string, colour string) string {
```

Comment:
Thanks for proposing this. I'm a bit uneasy about passing colour as string. I think it would be more interesting to use constants for that, as with a set of predefined constants you immediately know which colours are supported from the IDE code completion.

Reply:
I'm thinking something like:

(untested)
```go
type Colour int

const (
    BLACK Colour =  iota
    RED
    GREEN
    ...
)

var COLOURS = map[Colour]string {
    ...
}

func WithBackground(text string, colour Colour) {
   ...
}
```

Reply:
Thank you. It does makes sense. The suggested changes submitted. Thanks.